### DISCUSSION TITLE
Error when url has `hash url`

### DISCUSSION THREAD

**Discussion by rimzzlabs** [giscus/giscus#788] [Q&A] [answered]
So my website has id and anchor on every heading that url will be match as their id.

so when I initially enter a url for example: `https://url/blog/foo` this will works fine, however when url contains `hash url` for example `https://url/blog/foo#some-heading-tag` this will error to my Giscus, how to ignore the hash url ?

here is my giscus component, Im using nextjs

```tsx
      <Giscus
        emitMetadata='0'
        inputPosition='top'
        repo='********'
        repoId='********'
        category='General'
        categoryId='********'
        mapping='pathname'
        reactionsEnabled='0'
        loading='lazy'
      />
```

**Comment by laymonage** [comment, marked as answer]
What's the error? I don't see any errors here.

https://rizkicitra.dev/blog/how-to-setup-husky-nextjs#setup-lint-staged

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/6379424/201524878-aeb80c82-f773-42ee-9701-6fb8be3a7d5a.png">

**Reply by rimzzlabs** [reply]
ow wow, but yesterday I've got the error, it was like a broken iframe

I don't know how I came up with this, but thanks and sorry 😅

**Reply by laymonage** [reply]
No worries. Feel free to report back if you encounter it again and add more details about the error so I can look into it.
